src/v2/Apps/Order/Components/__tests__/AddressModal.jest.tsx

```diff
@@ -0,0 +1,172 @@
+import React from "react"
+import { commitMutation as _commitMutation } from "react-relay"
+import { AddressModal, Props, GENERIC_FAIL_MESSAGE } from "../AddressModal"
+import { mount } from "enzyme"
+import { validAddress } from "v2/Components/__tests__/Utils/addressForm"
+import {
+  updateAddressSuccess,
+  updateAddressFailure,
+} from "v2/Apps/Order/Routes/__fixtures__/MutationResults"
+import { SavedAddressType } from "../../Utils/shippingAddressUtils"
+
+const errorBoxQuery = "Text[data-test='credit-card-error']"
+
+// needed for modal contentAnimation
+const tick = () => new Promise(resolve => setTimeout(resolve, 0))
```

Comment:
What's this doing?

Reply:
ah this one took me a while to figure out. The modal has open/close/resize animations so this flushes those pending animations (took it from palette tests)